src/osd/osd_types.h

```diff
@@ -3792,6 +3809,7 @@ class pg_missing_const_i {
   virtual bool is_missing(const hobject_t& oid, pg_missing_item *out = nullptr) const = 0;
   virtual bool is_missing(const hobject_t& oid, eversion_t v) const = 0;
   virtual ~pg_missing_const_i() {}
+  virtual operator std::string() { return "needs implementation"; }
```

Comment:
Given that this class is already pure virtual, why not just make this pure virtual as well?

Reply:
You're obviously right. I was either not thinking, or this was a note to myself to look into implementing the operator in the base class. Either way, thanks!